### ISSUE TITLE
index.js is suspiciously obfuscated

### ISSUE THREAD

**Issue by chase-moskal** [kevinschaich/gapi-client#3]
i can't trust a library if i can't read its source code ;)

i also cannot contribute

**Comment by joshpitzalis** [comment]
That's just google's script, minified, untouched, copied and exported as an npm package. You could just use google's script directly. https://apis.google.com/js/api.js

**Comment by kevinschaich** [comment]
Yep, sorry for the obscurity @chase-moskal – @joshpitzalis is correct. This package is merely a wrapper around Google's JS client library. You are welcome to run a diff yourself to confirm.

Perhaps a better solution would be to automate this process (yarn/npm script to pull from Google's site, minify, and release an exported version).

I don't have time to do that at the moment but welcome contributions. In the absence of a contribution, you are free to use the library as-is or use another package. Thanks!
